src/Service.php

```diff
@@ -7,179 +8,315 @@
 namespace ZF\Apigility\Documentation\Swagger;
 
 use ZF\Apigility\Documentation\Service as BaseService;
+use ZF\Apigility\Documentation\Operation;
+use ZF\Apigility\Documentation\Field;
+use ZF\Apigility\Documentation\Swagger\Model\ModelGenerator;
 
 class Service extends BaseService
 {
+
+    const DEFAULT_TYPE = 'string';
+    const ARRAY_TYPE = 'array';
+
     /**
      * @var BaseService
      */
     protected $service;
+    protected $modelGenerator;
 
     /**
      * @param BaseService $service
-     * @param string $baseUrl
      */
-    public function __construct(BaseService $service, $baseUrl)
```

Comment:
Why do you remove the `$baseUrl` parameter? This is necessary when hosting the service under a separate directory of a primary website...

Reply:
It appears it was never used.

Reply:
Never mind; it was. The `ViewModel` class was creating `Service` instances previously, providing the baseUrl as a parameter, which was then used in the `basePath` element of the Swagger documentation. However, it appears this is no longer an element, which is why we can remove it here.